ckan/patches/ckan/unflattern_indexerror.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/ckan/lib/navl/dictization_functions.py b/ckan/lib/navl/dictization_functions.py
```

Comment:
Can you use `git format-patch` to create this diff instead? That way the commit and author information can be preserved as part of the patch.

Reply:
Updated [here](https://github.com/GSA/catalog.data.gov/pull/77/commits/58846679cd8f833a744d90c859e6829392489732)